### PR TITLE
provider/aws: Add a regression check for Route 53 records

### PR DESCRIPTION
**Pull request by catsby**
Regression Acceptance Test for #1122 cc @radeksimko

### CODE REVIEW

File: builtin/providers/aws/resource_aws_route53_record_test.go [hunk 1]
```diff
@@ -325,3 +349,26 @@ resource "aws_route53_record" "www-live" {
   records = ["dev.notexample.com"]
 }
 `
+
+const testAccRoute53RecordTypeChangePre = `
+
+resource "aws_route53_record" "sample" {
+	# zone_id = "${aws_route53_zone.main.zone_id}"
+  zone_id = "Z11E3X3FZHXR43"
```
Comment (catsby) [inline builtin/providers/aws/resource_aws_route53_record_test.go]: sigh...

**Comment by mitchellh** [comment]
LGTM

**Comment by radeksimko** [comment]
:+1:

**Comment by catsby** [comment]
Test failures here are unrelated; Acc tests pass, merging

**Pull request merged**